docs/modules/ROOT/pages/guides/contract_migration.adoc

```diff
@@ -0,0 +1,39 @@
+:source-highlighter: highlight.js
+:highlightjs-languages: rust
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+
+= Migrate a Contract from EVM chain
+
+This EVM template is powered by Frontier. It allows you to deploy almost any Solidity contract you have following these steps.
+
+== Step 1: Prepare and run the chain
+
+This step is basic, you can use our xref:index.adoc[guide for generic template]. It is recommended to use xref:guides/async_backing.adoc[async backing] for better experience.
+The only additional thing that you will need to specify is the EVM Chain Id in your chainspec.
```

Comment:
I believe you are referring to: https://github.com/polkadot-evm/frontier/blob/master/template/node/src/chain_spec.rs#L175C3-L175C41.
I think it will be better to explicitly provide steps on how to do it. Like:
1. go to node/src/chain_spec.rs
2. add this line to the bottom,
3. etc...

Reply:
Do they also need to add this line?
https://github.com/polkadot-evm/frontier/blob/master/template/node/src/chain_spec.rs#L176

Reply:
> Do they also need to add this line?
No, they don't. It is needed only for the fuzzer setup.

Reply:
Fixed the original message